### ISSUE TITLE
documentation on percentiles script

### ISSUE THREAD

**Issue by alexeyOnGitHub** [adamhadani/logtools#6]
I see the following output:

$ percentiles.sh -f "accessToken.txt"
Thu Oct 10 10:47:57 2010 - Number of samples used: 118684
Thu Oct 10 10:47:57 2010 - 10th Percentile (11868th Sample): 23
Thu Oct 10 10:47:57 2010 - 20th Percentile (23736th Sample): 40
Thu Oct 10 10:47:57 2010 - 30th Percentile (35605th Sample): 58
...
Thu Oct 10 10:47:59 2010 - 99th Percentile (117497th Sample): 1000 

my first guess is that "percentiles" code:
- opens the given file
- strips all data in all lines except for response time
- sorts the lines
- goes through the lines and prints response time for several "percentiles": 10,20,30, etc.

in other words, this shows that
10% of requests were completed in under 23 msec
20% of requests were completed in under 40 msec
etc
99% of requests were completed in under 1000 msec

is this correct? 
if yes, maybe it makes sense to update documentation to include this info?
